js/src/utilities/transform.lesson.js

```diff
@@ -20,10 +20,15 @@ const transformLesson = (node) => {
     return null;
   }
 
+  console.log(fields.getTextValue(node, 'title'));
```

Comment:
@esbaddeley I think we need to add a eslint rule that checks left out console.log messages ;)

Reply:
It actually already is a rule - but it gets lost in the 98 other errors when you run the linter :)